src/main/appMenuTemplate.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2020 Exotic Matter SAS. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for license information.
+ */
+
+import {app, shell} from 'electron';
+import {toggleDebugFileLogLevel} from "./index";
+
+const path = require('path');
+
+export function getTemplate(debugMode){
+  let template = [];
+  const isMac = process.platform === 'darwin';
+  const name = app.getName();
+  // OS X
+  if (isMac) {
+    template.push({
+      label: name,
+      role: 'appMenu'
+    });
+  }
+  template.push({
+    label: 'Advanced',
+    submenu: [
+      {
+        label: 'Enable DEBUG mode',
+        type: 'checkbox',
+        checked: debugMode,
+        click() { toggleDebugFileLogLevel(); }
+      },
+      {
+        label: 'Open logs folder',
+        click: () => {
+          let logsFolderPath;
+          if (isMac) {
+            logsFolderPath = path.join(app.getPath("home"), "Library", "Logs", "paper-matter-import-export");
+          } else {
+            logsFolderPath = path.join(app.getPath("appData"), "paper-matter-import-export", "logs");
```

Comment:
We should test this path is OK for Windows too (only tested under Linux)

Reply:
This is where the logs have been stored on my Windows machine.

![image](https://user-images.githubusercontent.com/10107072/98012458-6ac09400-1df9-11eb-987c-52b926474f67.png)

Reply:
This is the same folder opened when you hit `Alt` > menu **Advanced** > **Open logs folder** ?

Reply:
Yes.